src/Missings.jl

```diff
@@ -216,6 +216,6 @@ julia> passmissing((x,y)->"\$x \$y")(1, 2)
 julia> passmissing((x,y)->"\$x \$y")(missing)
 missing
 """
-passmissing(f::Base.Callable) = PassMissing{typeof(f)}(f)
+passmissing(f) = PassMissing{typeof(f)}(f)
```

Comment:
Unrelated to this PR, but I think `typeof(f)` here should actually be `Core.Typeof(f)`.
```julia
julia> typeof(Int)
DataType

julia> Core.Typeof(Int)
Type{Int64}

julia> typeof(sin) === Core.Typeof(sin)
true
```

Reply:
will open another PR :)

Reply:
(after this one is merged)

Reply:
Good point - I would propose to change this in this PR and add appropriate tests for: function, type and functor.

@nickrobinson251 - would you be willing to do this?

Reply:
👍

Reply:
Let me now if I missed anything 😄